handle_linux.go

```diff
@@ -61,10 +61,14 @@ func (h *Handle) SetSocketTimeout(to time.Duration) error {
 // SetSocketReceiveBufferSize sets the receive buffer size for each
 // socket in the netlink handle. The maximum value is capped by
 // /proc/sys/net/core/rmem_max.
-func (h *Handle) SetSocketReceiveBufferSize(size int) error {
+func (h *Handle) SetSocketReceiveBufferSize(size int, force bool) error {
```

Comment:
This is a breaking change for projects which use this API.
Please make this a variadic function instead:

`func (h *Handle) SetSocketReceiveBufferSize(size int, options... SocketOption) error {`

Reply:
This function was introduced in this same PR.

Reply:
Ah, I was looking at one commit's diff only. 👍

Reply:
This was put in a different commit because it makes the signature of the function a bit unfriendly. So, it's easy to remove this commit if we don't want it.